### DISCUSSION TITLE
The device inside the container is eth0, while on the host machine, it is enp3s0. WG_DEVICE is configured as enp3s0, but the traffic is not being forwarded. Can the network type of the container be set to host?

### DISCUSSION THREAD

**Discussion by iamckun** [wg-easy/wg-easy#742] [Q&A]
The device inside the container is eth0, while on the host machine, it is enp3s0. WG_DEVICE is configured as enp3s0, but the traffic is not being forwarded. Can the network type of the container be set to host?

**Comment by centralhardware** [comment]
Yes you can run container with network host but only one for now

**Reply by iamckun** [reply]
After using host, I didn't send any forwarding, so I changed it back to bridging
